### PR TITLE
Add Dutch translation.

### PR DESCRIPTION
**Pull request by JxxIT**
This adds an Dutch translation.

### CODE REVIEW

File: CONTRIBUTINGnl.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+#### Een functieverzoek implementeren
+
+Als je een functie hebt gevonden waarvan je denkt dat je die zou kunnen implementeren, wees dan niet verlegen. Open een PR met de oplossing en zorg ervoor dat u de functie vermeldt die u implementeert.
+
+### Welke PR's accepteren we?
+
+- Probleemoplossingen.
+- Implementatie van functies.
+- Typfouten of betere en gemakkelijkere woorden om te gebruiken.
```
Comment (YassineGherbi) [inline CONTRIBUTINGnl.md:66]: Ironic haha. Het is "Typefouten" ipv "Typfouten" :)

File: Docs/FAQnl.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### **3. Hoe werkt dit?**
+
+De extensie verzamelt de video-ID van de video die je aan het bekijken bent, haalt de dislike (en andere velden zoals views, likes etc) op met behulp van onze API. Als dit de eerste keer is dat de video wordt opgehaald door onze API, zal deze de YouTube API gebruiken om de gegevens op te halen, slaat de gegevens vervolgens op in een database voor caching (cache voor ongeveer 2-3 dagen) en archiveringsdoeleinden en stuurt ze naar u terug. De extensie geeft vervolgens de antipathieën aan u weer.
+
+<br>
+
+### **4. Wat gebeurt er nadat de YouTube API stopt met het teruggeven van het aantal dislikes?**
+
+De backend zal overschakelen naar het gebruik van een combinatie van gearchiveerde afkeerstatistieken, schattingen geëxtrapoleerd uit gebruikersgegevens van extensies en schattingen op basis van weergave/vind-ik-leuk-verhoudingen voor video's waarvan de afkeuren niet zijn gearchiveerd en voor verouderde afkeer-archieven.
```
Comment (YassineGherbi) [inline Docs/FAQnl.md:33]: Correct spelling should be "afkeer statistieken" en "afkeer archieven" if I am really 'aan het mierenneuken'

File: Docs/FAQnl.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+[![IReturn YouTube Dislike Uitgelegd (Engels)](https://yt-embed.herokuapp.com/embed?v=GSmmtv-0yYQ)](https://www.youtube.com/watch?v=GSmmtv-0yYQ)
+
+---
+
+<br>
+
+## Ik heb zorgen over beveiliging/privacy
+
+See [this page](SECURITY-FAQ.md) for more info.
```
Comment (YassineGherbi) [inline Docs/FAQnl.md:61]: Maybe we could also translate this sentence? Or did you skip it for a reason?

**Review by YassineGherbi — CHANGES_REQUESTED** [review]
2 small spelling mistakes but awesome work! Do we really need 6 approvals? Seems like to lot to me, tbh. @Anarios

**Comment by JxxIT** [comment]
If fixed the typos.

**Review by OpenSourceSimon — APPROVED** [review]

**Review by YassineGherbi — APPROVED** [review]

**Comment by YassineGherbi** [comment]
I would suggest tagging some people who could also review. You'll need 5 more approving reviews to be able to merge this automatically

**Comment by JxxIT** [comment]
Yes, but who can approve it?

**Pull request merged**